release.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+# Usage: $ ./prepare_for_release.py <type>
+# type - [major | minor | patch]. Default is patch.
```

Comment:
@wslulciuc, you mentioned that you had a preference for the usage statement. Would you suggest any changes to what I have here?